tests/phpunit/tests/Plugin.php

```diff
@@ -47,7 +47,6 @@ public function test_register() {
 	 * @preserveGlobalState disabled
 	 */
 	public function test_is_amp_endpoint_polyfill_on_wp_action() {
-		$this->assertFalse( function_exists( '\is_amp_endpoint' ) );
```

Comment:
Can you explain this change?

Reply:
Probably because the function is now only added on `init`, not `wp` anymore. But then the test doesn't really make sense anymore in this form.

Reply:
I think this change is not needed as of now in this PR, but this will go into some other PR where required. Reverting this change.

Reply:
Same as [here](https://github.com/rtCamp/web-stories-wp/pull/189#discussion_r552395182)

Reply:
Agree with @swissspidy , we can remove the test as this doesn't make sense in this form.

Reply:
Can probably leave this file unchanged now that we have this new `is_amp()` function